builtins/statements/set.go

```diff
@@ -52,15 +52,19 @@ func (stmt *SetStmt) Execute(r *exec.Renderer, tag *nodes.StatementBlock) error
 		if target.IsError() {
 			return errors.Wrapf(target, `Unable to evaluate target %s`, n)
 		}
-		if err := target.Set(n.Attr, value.Interface()); err != nil {
+		if n.Attr == "" {
+			return errors.Errorf(`Not implemented to evaluate getattr at %s`, n.Attr) // TODO: implement
```

Comment:
`...%s, n.Attr...` not really useful. n.Attr is always empty

Reply:
actual it's printing the wrong thing: https://github.com/paradime-io/gonja/pull/1/commits/5f821f23d18e8f41fd9d76896dac5158cf0977f4

Reply:
well spotted!